src/System.Management.Automation/resources/FormatStrings.Designer.cs

```diff
@@ -0,0 +1,108 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace System.Management.Automation.resources {
```

Comment:
The file should be generated to /gen folder by `Start-PSBuild -ResGen`.

Reply:
Whoops, that was my mistake. Corrected and committed. Thanks.